release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -78,8 +78,159 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-ibm-z"]
 === IBM Z and LinuxONE
 
+With this release, IBM Z and LinuxONE are now compatible with {product-title} {product-version}. The installation can be performed with z/VM or {op-system-base} KVM. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE in a restricted network]
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Installing a cluster with {op-system-base} KVM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc#installing-restricted-networks-ibm-z-kvm[Installing a cluster with RHEL KVM on IBM Z and LinuxONE in a restricted network]
+
+[discrete]
+==== Notable enhancements
+
+The following new features are supported on IBM Z and LinuxONE with {product-title} {product-version}:
+
+* Horizontal pod autoscaling
+* The following Multus CNI plug-ins are supported: 
+** Bridge
+** Host-device
+** IPAM
+** IPVLAN
+
+* NMState Operator
+* OVN-Kubernetes IPsec encryption
+* Vertical Pod Autoscaler Operator
+
+[discrete]
+==== Supported features
+
+The following features are also supported on IBM Z and LinuxONE:
+
+* Currently, the following Operators are supported:
+** Cluster Logging Operator
+** Local Storage Operator
+** NFD Operator
+** NMState Operator
+** OpenShift Elasticsearch Operator
+** Service Binding Operator
+** Vertical Pod Autoscaler Operator
+
+* Encrypting data stored in etcd
+* Helm 
+* Multipathing
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* Persistent storage using hostPath
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* OVN-Kubernetes
+* Support for multiple network interfaces
+* Three-node cluster support
+* z/VM Emulated FBA devices on SCSI disks
+* 4K FCP block device
+
+These features are available only for {product-title} on IBM Z and LinuxONE for {product-version}:
+
+* HyperPAV enabled on IBM Z and LinuxONE for the virtual machines for FICON attached ECKD storage
+
+[discrete]
+==== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
```

Comment:
Could we reword this to omit "note"? Perhaps: 
The following restrictions impact {product-title} on IBM Z and LinuxONE: